boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts

```diff
@@ -19,6 +19,7 @@
 		led0 = &seg7_led1_g;
 		led1 = &seg7_led2_g;
 		sw0 = &user_button1;
+		eeprom-0= &eeprom;
```

Comment:
Nitpick: add missing space before `=`, i.e. `eeprom-0 = &eeprom;`

Reply:
Thanks, fixed it.